### ISSUE TITLE
Can we use HDF5 Format to read Image and labels for training Image using YOLOV5?

### ISSUE THREAD

**Issue by karndeepsingh** [ultralytics/yolov5#4108]
## ❔Question
Hi,
Can we use HDF5 Format to read Image and labels for training Image using YOLOV5? As I am trying to run this YOLOV5 model on azure cloud and my team is trying to store annotated image in HDF5  formate. Will that work? or we need pure JPG or PNG files for training.

**Comment by glenn-jocher** [comment]
@karndeepsingh can you provide supporting information, links, references etc for your question?

**Comment by glenn-jocher** [comment]
@karndeepsingh you may also want to talk to this contributor: https://github.com/ultralytics/yolov5/pull/4049

**Comment by karndeepsingh** [comment]
@glenn-jocher  Actually, my team is going to use Databricks platform o train YOLOV5 object detection model. Till now we were training it on locally with less image numbers. But now we are going to train with 100K images with annotation on DataBricks. And they said it stores data in HDF5 format. So,training on Databrick will progress when I use HDF5 format or shall I store images in Azure Blob storage and connect with DataBricks and then load images in PNG OR JPG format from blob storage and train.

Any help  and advise would be good.
Thanks

**Comment by glenn-jocher** [comment]
@karndeepsingh I don't know about other dataset formats, but you can see many examples of YOLOv5 auto-download datasets just by requesting a dataset and letting it download i.e.:
``` 
python train.py --data VOC.yaml
python train.py --data VisDrone.yaml
python train.py --data Objects365.yaml
etc...
```

See data directory for more.

**Comment by karndeepsingh** [comment]
@glenn-jocher Thanks for adding the information. I had doubt that whether we will be able to train YOLOV5 model on Databricks or not.

Please provide us help on this knowledge as well.

**Comment by glenn-jocher** [comment]
@karndeepsingh for 3rd party YOLOv5 support you should reach out directly to those authors or organizations.

The standard dataset format is described in our Train Custom Data Tutorial below:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
